apps/react-18-tests-v9/jest.config.js

```diff
@@ -1,7 +1,24 @@
 // @ts-check
+/* eslint-disable */
 
+const { readFileSync } = require('node:fs');
 const { join } = require('node:path');
 
+// Reading the SWC compilation config and remove the "exclude"
+// for the test files to be compiled by SWC
+const { exclude: _, ...swcJestConfig } = JSON.parse(readFileSync(join(__dirname, '.swcrc'), 'utf-8'));
```

Comment:
added swc processing to properly run tests in v9 projects - same config as we have in v9 libraries